app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,56 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .dev-tag{
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Opening curly brace `{` should be preceded by one space